website/docs/r/security_tag.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: security_tag"
+sidebar_current: "docs-vcd-resource-security-tag"
+description: |-
+Provides a VMware Cloud Director Security Tag resource. This can be
+used to assign security tag to VMs.
+---
+
+# vcd\_security\_tag
+
+Provides a VMware Cloud Director Security Tag resource. This can be
+used to assign security tag to VMs.
+
+Supported in provider *v3.7+* and requires VCD 10.3.1+
+
+-> **Note:** This resource requires either system or org administrator privileges.
+
+## Example Usage
+
+```hcl
+resource "vcd_security_tag" "my_tag" {
+  name = "test-tag"
+  vm_ids = [vcd_vm.my-vm-one.id, vcd_vm.my-vm-two.id]
+}
+```
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the security tag.
+* `vm_ids` - (Required) List of VM IDs that the security tag is going to be applied to.
+
```

Comment:
We don't usually put `## Attribute ` section, but could you just mention here that the `id` of `vcd_security_tag` is set to `name` because no real ID for security tag exists.

Reply:
Add a comment about it